assemblies/prd-ce/src/main/resources-filtered/report-designer.sh

```diff
@@ -48,6 +48,7 @@ JAVA_ADD_OPENS="$JAVA_ADD_OPENS --add-opens=java.base/java.math=ALL-UNNAMED"
 JAVA_ADD_OPENS="$JAVA_ADD_OPENS --add-opens=java.base/sun.nio.ch=ALL-UNNAMED"
 JAVA_ADD_OPENS="$JAVA_ADD_OPENS --add-opens=java.base/java.nio=ALL-UNNAMED"
 JAVA_ADD_OPENS="$JAVA_ADD_OPENS --add-opens=java.security.jgss/sun.security.krb5=ALL-UNNAMED"
+JAVA_ADD_OPENS="$JAVA_ADD_OPENS --add-opens=java.desktop/com.apple.eawt=ALL-UNNAMED"
```

Comment:
I think this needs to get moved inside of the block for darwin OS below; on other systems this results in a warning message on startup:

WARNING: package com.apple.eawt not in java.desktop